atomate/vasp/firetasks/write_inputs.py

```diff
@@ -60,6 +61,45 @@ def run_task(self, fw_spec):
             vis = vis_cls(self["structure"], **self.get("vasp_input_params", {}))
         vis.write_input(".")
 
+@explicit_serialize
+class WriteVaspFromIOSetFromInterpolatedPOSCAR(GetInterpolatedPOSCAR):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
+    structure. Create VASP input files using implementations of pymatgen's
+    AbstractVaspInputSet. An input set can be provided as an object or as
+    a String/parameter combo.
+
+    Required params:
+        start (str): name of fw for start of interpolation.
+        end (str): name of fw for end of interpolation.
+        this_image (int): which interpolation this is.
+        nimages (int) : number of interpolations.
+        autosort_tol (float): a distance tolerance in angstrom in which
+          to automatically sort end_structure to match to the closest
+          points in this particular structure.
+        vasp_input_set (str): a string name for the VASP input set (e.g., "MPRelaxSet").
+
+    Optional params:
+        vasp_input_params (dict): When using a string name for VASP input set, use this as a dict
+            to specify kwargs for instantiating the input set parameters. For example, if you want
+            to change the user_incar_settings, you should provide: {"user_incar_settings": ...}.
+            This setting is ignored if you provide the full object representation of a VaspInputSet
+            rather than a String.
+    """
+    # First, we make a fresh copy of the required_params before modifying.
+    required_params = list(GetInterpolatedPOSCAR.required_params)
```

Comment:
I can see what you're doing here, but I think you should just set the required_params explicitly rather than try to extend required_parameters from another class. Hopefully the unit tests will fail if the required_params for GetInterpolatedPOSCAR change, so that we won't have mismatch between required parameters. I